cmd/status-backend/main.go

```diff
@@ -9,14 +9,16 @@ import (
 
 	"github.com/ethereum/go-ethereum/log"
 
-	"github.com/status-im/status-go/cmd/status-backend/server"
+	backendServer "github.com/status-im/status-go/cmd/status-backend/server"
 	"github.com/status-im/status-go/internal/version"
 	"github.com/status-im/status-go/logutils"
+	mediaServer "github.com/status-im/status-go/server"
 )
 
 var (
-	address = flag.String("address", "", "host:port to listen")
-	logger  = log.New("package", "status-go/cmd/status-backend")
+	address       = flag.String("address", "", "host:port to listen")
+	useMediaHTTPS = flag.Bool("media-https", true, "use HTTPS for media server (default: true)")
```

Comment:
This setting has nothing to do with `status-backend`, which only provides full API through HTTP.
It doesn't start the media server, hence, it shouldn't provide controls for the media server configuration.

If HTTPS has to be disabled, then it should be done through some API.

Reply:
If you set it through the API then it should be simpler to avoid the existence of `server.UseHTTP` global variable.

Reply:
> Regarding rest comments you wrote, can I address it in a followup PR?

Sure

Reply:
> The key reason is not relate to the parameter MediaServerEnableTLS, it's not possible for mobile frontend update the cert okhttpclient used which comes from built-in status-go function ImageServerTLSCert

@qfrank I'm not arguing for removing this parameter completely. I'm only asking to set the `server.UseHTTPS` not through `status-backend --media-https` CLI option, but with a `requests.InitializeApplication. MediaServerEnableTLS` flag instead 🙂

Reply:
As discussed in DM, using `InitializeApplication` is possible.

To be clear, I insist on API solution because CLI option is against the simple `status-backend` idea: provide status-go API with HTTP.

It must do nothing else but be an HTTP server. And if you start adding more options, you'll end up with `statusd` with tons of CLI flags for each edge case.